docs/contributing/database-migrations/edd.mdx

```diff
@@ -32,73 +29,76 @@ For background on this decision please see the [Evolutionary Database Design RFD
 
 ## Design
 
-Database changes can be categorized into two categories: destructive and non-destructive
+Database changes can be categorized into two categories: destructive and non-destructive changes
 \[[1](./edd#further-reading)\]. A destructive change prevents existing functionality from working as
```

Comment:
🎨 Can we link this to wording vs. a reference tag? Could link "two categories".

Reply:
It feels more like a foot note since the categories are denoted below and a link could be presumed to go to the categories within the document.